### PR TITLE
[EDU-4205] feat: add ref on s3 protocol and guide on s3cmd

### PR DESCRIPTION
**Pull request by bru-andrade**
### Changes

- Adds new sections to doc on credentials and s3 protocol
- Adds new guide on how to create credentials and use s3cmd 
- Minor changes: operation name fixes, remove `Azion` as part of Edge Storage name

### Additional links

[EDU-3000]: https://aziontech.atlassian.net/browse/EDU-3000?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[EDU-4205]: https://aziontech.atlassian.net/browse/EDU-4205?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by gabriel-azion — APPROVED** [review]

File: src/content/docs/en/pages/main-menu/reference/store/object-storage/index.mdx [hunk 1]
```diff
@@ -99,90 +85,164 @@ You can determine if the content is retrieved from the root of the bucket or fro
...
 
-Use the `--data-binary` option in the cURL command to send raw binary data, such as images or complex files without undergoing URL encoding.
+:::tip
+For API cURL requests, use the `--data-binary` option in the command to send raw binary data, such as images or complex files without undergoing URL encoding.
+:::
 
-You must specify the MIME type of the object being sent in the body using the `Content-Type` header. For example, objects with the `.txt` extension should contain the `Content-Type: text/plain` header. Alternatively, use the `application/octet-stream` MIME type to indicate that the data is a binary stream and the server should handle it as raw binary data.
+For the **Azion API**, you can specify the MIME type of the object being sent in the body using the `Content-Type` header. For example, objects with the `.txt` extension should contain the `Content-Type: text/plain` header. If the MIME type is not specified, **Edge Storage** will attempt to interpret the file type based on the file extension. Alternatively, use the `application/octet-stream` MIME type to indicate that the data is a binary stream and the server should handle it as raw binary data.
```
Comment (hannah-sk) [inline src/content/docs/en/pages/main-menu/reference/store/object-storage/index.mdx:120]:
```suggestion
For the **Azion API**, you can specify the MIME type of the object being sent in the body using the `Content-Type` header. For example, objects with the `.txt` extension should contain the `Content-Type: text/plain` header. If the MIME type isn't specified, **Edge Storage** will attempt to interpret the file type based on the file extension. Alternatively, use the `application/octet-stream` MIME type to indicate that the data is a binary stream and the server should handle it as raw binary data.
```

File: src/content/docs/en/pages/main-menu/reference/store/object-storage/index.mdx [hunk 2]
```diff
@@ -99,90 +85,164 @@ You can determine if the content is retrieved from the root of the bucket or fro
...
+- **Read-write**: objects in the bucket can be modified.
+- **Restricted**: objects in the bucket can be modified and read but can't be accessed by the Azion Edge Platform. Buckets can't be modified using Edge Runtime and can't be used as an origin for edge applications.
+
+### S3 credentials
+
+**Edge Storage** offers compatibility with the S3 protocol through credentials. 
+
+Credentials can be created for any bucket that you own or for your account as a whole. With them, you can control permissions for operations associated with that credential. The permissions for the credential are exclusive to access thorugh the S3 protocol.
+
+Once a credential is created, an *access key* and a *secret key* are generated, which can be used to set up access to the bucket through the S3 protoco. For security reasons, the *secret key* won't be available after the credential is created. Existing credentials cannot be modified in any way.
```
Comment (hannah-sk) [inline src/content/docs/en/pages/main-menu/reference/store/object-storage/index.mdx:207]:
```suggestion
Once a credential is created, an *access key* and a *secret key* are generated, which can be used to set up access to the bucket through the S3 protoco. For security reasons, the *secret key* won't be available after the credential is created. Existing credentials can't be modified in any way.
```

File: src/content/docs/en/pages/main-menu/reference/store/object-storage/index.mdx [hunk 3]
```diff
@@ -99,90 +85,164 @@ You can determine if the content is retrieved from the root of the bucket or fro
...
+
+Once a user's access is verified, they are allowed to make operations depending on the capabilities set for the credential.
```
Comment (hannah-sk) [inline src/content/docs/en/pages/main-menu/reference/store/object-storage/index.mdx:209]:
```suggestion
Once a user's access is verified, they're allowed to make operations depending on the capabilities set for the credential. 
```

File: src/content/docs/pt-br/pages/guias/object-storage/use-s3-protocol/index.mdx [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+permalink: /documentacao/produtos/guias/protocolo-s3-para-edge-storage/
+---
+
+O **Edge Storage** da Azion oferece compatibilidade com o protocolo S3 por meio de credenciais. Quando você cria uma credencial para qualquer bucket que você possui, você receberá uma access key e uma secret key para configurar permissões para operações. O acesso ao seu bucket por meio do protocolo S3 será verificado usando a credencial.
+
+O protocolo S3 permite que você acesse buckets e objetos usando uma URL do Edge Storage. Esta configuração facilita operações de arquivo por meio de ferramentas de interface de linha de comando (CLI), como [s3cmd](https://s3cmd.org), serviços de banco de dados ou funções.
+
+---
+
+## Criar uma credencial via API
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/guias/object-storage/use-s3-protocol/index.mdx:15]:
```suggestion
## Crie uma credencial via API
```

File: src/content/docs/pt-br/pages/guias/object-storage/use-s3-protocol/index.mdx [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+        "created_at": "<datetime>"
+    }
+}
+```
+
+3. Copie os valores de `access_key` e `secret_key` para configurar o acesso através do protocolo S3.
+
+---
+
+## Configurar o acesso ao bucket através do s3cmd
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/guias/object-storage/use-s3-protocol/index.mdx:74]:
```suggestion
## Configure o acesso ao bucket através do s3cmd
```

File: src/content/docs/en/pages/main-menu/reference/store/object-storage/index.mdx [hunk 6]
```diff
@@ -99,90 +85,164 @@ You can determine if the content is retrieved from the root of the bucket or fro
...
+
+To do so, you'll need the following information:
+
+| Data | Description |
+| --- | --- |
+| Access key | The credential's access key generated upon creating the S3 credential with the Azion API. |
+| Secret key | The credential's secret key generated upon creating the S3 credential with the Azion API. This information is confidential and will only be available at the moment of creation. |
+| Region | The assigned server's region, which is `us-east-005`. |
+| S3 endpoint | The default S3 address for all operations, which is `s3.us-east-005.azionstorage.net`. |
+| DNS-style template | The host name template to access the bucket and objects. Can be `bucket+hostname:port/object-key` or `hostname:port/bucket`. <br /><br />For example, for a `file.txt` object in the `my-bucket` bucket, the host names could be: <br /><li><ul>`my-bucket.s3.us-east-005.azionstorage.net/file.txt`</ul><ul>`s3.us-east-005.azionstorage.net/my-bucket/file.txt`</ul></li> |
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/main-menu/reference/store/object-storage/index.mdx:242]:
```suggestion
| Access key | The credential's access key generated upon creating the S3 credential with the Azion API |
| Secret key | The credential's secret key generated upon creating the S3 credential with the Azion API. This information is confidential and will only be available at the moment of creation |
| Region | The assigned server's region, which is `us-east-005` |
| S3 endpoint | The default S3 address for all operations, which is `s3.us-east-005.azionstorage.net` |
| DNS-style template | The host name template to access the bucket and objects. Can be `bucket+hostname:port/object-key` or `hostname:port/bucket`. <br /><br />For example, for a `file.txt` object in the `my-bucket` bucket, the host names could be: <br /><li><ul>`my-bucket.s3.us-east-005.azionstorage.net/file.txt`</ul><ul>`s3.us-east-005.azionstorage.net/my-bucket/file.txt`</ul></li> |
```

File: src/content/docs/pt-br/pages/menu-principal/referencia/store/object-storage/index.mdx [hunk 7]
```diff
@@ -99,90 +85,164 @@ Você pode determinar se o conteúdo é retirado da raiz do bucket ou de um pref
...
+:::tip
+Para requests feitos pela API com o comando cURL, utilize a opção `--data-binary` para enviar dados binários brutos, como imagens ou arquivos complexos sem passar por URL encoding.
+:::
 
-Você deve especificar o MIME type do objeto sendo enviado no corpo da requisição com o cabeçalho `Content-Type`. Por exemplo, objetos com extensão `.txt` devem conter o cabeçalho `Content-Type: text/plain`. Como alternativa, utilize o MIME type `application/octet-stream` para indicar que os dados são um fluxo binário e o servidor deve tratá-los como dados binários brutos.
+Com a **API da Azion**, você pode especificar o MIME type do objeto sendo enviado no corpo da requisição com o cabeçalho `Content-Type`. Por exemplo, objetos com extensão `.txt` devem conter o cabeçalho `Content-Type: text/plain`.Se o MIME type não for especificado, o **Edge Storage** tentará interpretar o tipo de arquivo com base na extensão do arquivo. Como alternativa, utilize o MIME type `application/octet-stream` para indicar que os dados são um fluxo binário e o servidor deve tratá-los como dados binários brutos.
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/menu-principal/referencia/store/object-storage/index.mdx:120]:
```suggestion
Com a **API da Azion**, você pode especificar o MIME type do objeto sendo enviado no corpo da requisição com o cabeçalho `Content-Type`. Por exemplo, objetos com extensão `.txt` devem conter o cabeçalho `Content-Type: text/plain`. Se o MIME type não for especificado, o **Edge Storage** tentará interpretar o tipo de arquivo com base na extensão do arquivo. Como alternativa, utilize o MIME type `application/octet-stream` para indicar que os dados são um fluxo binário e o servidor deve tratá-los como dados binários brutos.
```

File: src/content/docs/pt-br/pages/menu-principal/referencia/store/object-storage/index.mdx [hunk 8]
```diff
@@ -99,90 +85,164 @@ Você pode determinar se o conteúdo é retirado da raiz do bucket ou de um pref
...
+
+Ações envolvendo buckets, como fazer upload, listar e excluir objetos, sempre exigem autenticação pela conta Azion.
+
+:::note
+Consulte [Teams Permissions](/pt-br/documentacao/produtos/gestao-de-contas/teams-permissions/) para saber mais sobre as permissões de contas do Edge Storage.
+:::
+
+Você pode gerenciar dois tipos de permissões para seus buckets e objetos:
+
+- **Permissões de bucket**: gerencie o acesso do edge e de outros usuários aos buckets e objetos dentro dos buckets usando a API REST da Azion.
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/menu-principal/referencia/store/object-storage/index.mdx:186]:
```suggestion
- **Permissões de bucket**: gerencie o acesso do edge e de outros usuários aos buckets e objetos dentro dos buckets usando a API da Azion.
```

File: src/content/docs/pt-br/pages/guias/object-storage/use-s3-protocol/index.mdx [hunk 9]
```diff
@@ -0,0 +1,104 @@
...
+
+[s3cmd](https://s3cmd.org) é uma ferramenta de interface de linha de comando (CLI) para gerenciar o S3 e outros serviços de armazenamento em nuvem. Pode ser usada para gerenciar objetos no **Edge Storage** e outros serviços de armazenamento compatíveis com S3.
+
+Para usar o s3cmd para gerenciar seu bucket no Edge Storage, siga estes passos:
+
+1. Baixe e instale o pactoe s3cmd através do [site oficial](https://s3cmd.org/download).
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/guias/object-storage/use-s3-protocol/index.mdx:80]:
```suggestion
1. Baixe e instale o pacote s3cmd através do [site oficial](https://s3cmd.org/download).
```

File: src/content/docs/pt-br/pages/menu-principal/referencia/store/object-storage/index.mdx [hunk 10]
```diff
@@ -99,90 +85,164 @@ Você pode determinar se o conteúdo é retirado da raiz do bucket ou de um pref
...
+<Button href="/pt-br/documentacao/produtos/guias/protocolo-s3-para-edge-storage/" text="saiba Como acessar o Edge Storage usando o protocolo S3" variant="secondary"/>
+
+Para fazer isso, você precisará das seguintes informações:
+
+| Dado | Descrição |
+| --- | --- |
+| Access key | A access key da credencial gerada ao criar a credencial S3 com a API da Azion. |
+| Secret key | A secret key da credencial gerada ao criar a credencial S3 com a API da Azion. Esta informação é confidencial e só estará disponível no momento da criação. |
+| Região | A região do servidor atribuída, que é `us-east-005`. |
+| Endpoint S3 | O endereço S3 padrão para todas as operações, que é `s3.us-east-005.azionstorage.net`. |
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/menu-principal/referencia/store/object-storage/index.mdx:241]:
```suggestion
| Access key | A access key da credencial gerada ao criar a credencial S3 com a API da Azion |
| Secret key | A secret key da credencial gerada ao criar a credencial S3 com a API da Azion. Esta informação é confidencial e só estará disponível no momento da criação |
| Região | A região do servidor atribuída, que é `us-east-005` |
| Endpoint S3 | O endereço S3 padrão para todas as operações, que é `s3.us-east-005.azionstorage.net` |
```

**Review by evandroazion — APPROVED** [review]

File: src/content/docs/en/pages/guides/object-storage/use-s3-protocol/index.mdx [hunk 11]
```diff
@@ -0,0 +1,104 @@
...
+
+To use s3cmd to manage your Edge Storage bucket, follow these steps:
+
+1. Download and install s3cmd package through the [official website](https://s3cmd.org/download).
+2. Make sure `s3cmd` is added to your system's PATH.
+3. Run `s3cmd --configure` and enter the access key and secret key.
+4. Enter the default region for the Edge Storage region: `us-east-005`.
+5. Enter the endpoint URL for Edge Storage: `s3.us-east-005.azionstorage.net`.
+6. Use the default DNS template: `%(bucket)s.s3.amazonaws.com`.
+7. Inform an encryption password and a path to GPG program if needed.
```
[the remark this answers is not in the document]
Comment (bru-andrade) [reply src/content/docs/en/pages/guides/object-storage/use-s3-protocol/index.mdx:86]: Não tem mas posso adicionar aqui :)

Comment (bru-andrade) [inline src/content/docs/en/pages/guides/object-storage/use-s3-protocol/index.mdx:86]:
```suggestion
7. Inform an encryption password and a path to GPG program if needed.
:::note
Gnu Privacy Guard (GPG) is a free and open-source encryption software that provides an additional layer cryptographic privacy and authentication for S3 protocol communication. See [the GnuPG documentation](https://www.gnupg.org/index.html) for more information.
:::

```

File: src/content/docs/pt-br/pages/guias/object-storage/use-s3-protocol/index.mdx [hunk 12]
```diff
@@ -0,0 +1,104 @@
...
+
+Para usar o s3cmd para gerenciar seu bucket no Edge Storage, siga estes passos:
+
+1. Baixe e instale o pacote s3cmd através do [site oficial](https://s3cmd.org/download).
+2. Certifique-se de que o `s3cmd` esteja adicionado ao PATH do seu sistema.
+3. Execute `s3cmd --configure` e insira a access key e a secret key.
+4. Insira a região padrão para a região do Edge Storage: `us-east-005`.
+5. Insira a URL do endpoint para o Edge Storage: `s3.us-east-005.azionstorage.net`.
+6. Use o template DNS padrão: `%(bucket)s.s3.amazonaws.com`.
+7. Informe uma senha de criptografia e um caminho para o programa GPG, se necessário.
```
Comment (bru-andrade) [inline src/content/docs/pt-br/pages/guias/object-storage/use-s3-protocol/index.mdx:86]:
```suggestion
7. Informe uma senha de criptografia e um caminho para um programa GPG, se necessário.

:::note
O Gnu Privacy Guard (GPG) é um software de criptografia gratuito e de código aberto que fornece uma camada adicional de privacidade criptográfica e autenticação para comunicação por protocolo S3. Consulte a [documentação do GnuPG](https://www.gnupg.org/index.html) para mais informações.
:::
```

**Pull request merged**